gh-pages/_posts/how-to/2020-08-17-fixate-folders-with-a-custom-cc-json.md

```diff
@@ -0,0 +1,98 @@
+---
+categories:
+    - How-to
+title: Fixate folders using a custom cc.json
+---
+
+# Introduction
+
+The [Squarified-Tree-Map-Algorithm](https://www.win.tue.nl/~vanwijk/stm.pdf) generates a layout for the visualized map.
+The layout varies depending on the area of each node. Adding a new file to a folder might increase the area of the parent folder,
+so that the algorithm decides that the parent-folder needs to be relocated. Another way to follow this problem is by changing the area-metric.
+Notice, that most folders will not be located at the same position as before.
+
+# Fixate folders
+
+In order to prevent folders from changing locations, we introduced a new attribute to the `.cc.json` called `fixed`.
+This attribute will not be generated during the analysis. The user needs to edit the `.cc.json` by hand.
+
+# API
+
+```json
+ "fixed": {
+    "x": 60,
+    "y:": 40,
+    "width": 35,
+    "height": 55
+ }
+```
+
+The `fixed`-attribute includes a coordinate `x` and `y`. These represent the top-left corner of the folder. `width`
+defines the length in x-direction and `height` defines the length in y-direction.
```

Comment:
We might not need this if we rename the properties to something more intuitive?

Reply:
True. Should we keep `width` and `height` or should we use `width` and `length`?

Reply:
We should keep `height`.